browser-test/src/admin_program_creation.test.ts

```diff
@@ -126,6 +125,43 @@ describe('program creation', () => {
     await endSession(browser)
   })
 
+  it('create question from question bank', async () => {
+    const {browser, page} = await startSession()
+
+    await loginAsAdmin(page)
+    const adminQuestions = new AdminQuestions(page)
+    const adminPrograms = new AdminPrograms(page)
+    const programName = 'apc program 3'
+    await adminPrograms.addProgram(programName)
+    await adminPrograms.goToManageQuestionsPage(programName)
+    await page.click('#create-question-button')
+    await page.click('#create-text-question')
+    await waitForPageJsLoad(page)
+
+    const [questionName, questionText] = [
+      'new-from-question-bank',
+      'Question text',
+    ]
```

Comment:
That looks a bit odd. 
```
const questionName = ...;
const questionText = ...;
```
is not much longer and easier to read.

Reply:
Indeed. Prior to formatting, this was all on one line, which made it a bit easier to read. I agree that the formatting made this more confusing than helpful. Switched to two separate initializations.